frontend/test/metabase/scenarios/collections/revision-history.cy.spec.js

```diff
@@ -42,27 +42,6 @@ describe("revision history", () => {
 
       cy.findAllByText("Revert").should("not.exist");
     });
-
-    it.skip("dashboard should update properly on revert (metabase#6884)", () => {
```

Comment:
This repro for the issue ([commit](https://github.com/metabase/metabase/commit/1e1b73e216f54a1a3fe313a7448f6725da7615ec)) is not relevant for the solution. Adding a card, as this does, shouldn't allow the user to revert to the "added a card" revision in the first place. Instead, after adding a card, the last revision should be "added a card", so it shouldn't be possible to revert to it. 

e.g. note the lack of a revert button on the "You added a card" revision
<img width="304" alt="image" src="https://user-images.githubusercontent.com/39073188/212770172-169300e2-9169-46dc-ae8a-61c264725fae.png">